candump.c

```diff
@@ -383,22 +388,37 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
-	for (i=0; i < currmax; i++) {
+	memset(&event_setup, 0, sizeof(event_setup));
```

Comment:
please use C99 initializers instead of memset + assignment

Reply:
memset() removed, it is not necessary and was just meant to safeguard.

Reply:
the `memset()` is not needed, if you use C99 initializers, see next review for an exmaple.